mobile/src/components/common/text/text.tsx

```diff
@@ -6,7 +6,7 @@ import { AppTextStyle } from '~/common/types/types';
 import { styles } from './styles';
 
 type Props = {
-  children: string;
+  children: string | React.ReactNode;
```

Comment:
Do we need this? let's leave just string for now
```suggestion
  children: string;
```